jvm/core/src/main/java/ml/dmlc/tvm/NDArrayBase.java

```diff
@@ -57,8 +57,7 @@ public NDArrayBase copyTo(NDArrayBase target) {
   /**
    * Release the NDArray memory.
    * <p>
-   * We highly recommend you to do this manually since the GC strategy is lazy
-   * and `finalize()` is not guaranteed to be called when GC happens.
```

Comment:
Is there a particular reason why this has been deleted? This info might be useful to keep.

Reply:
actually finalize() is guaranteed to be called finally. it just cannot be called immediately.